front/app/components/ConsentManager/index.tsx

```diff
@@ -186,6 +186,26 @@ export class ConsentManager extends PureComponent<Props, State> {
     );
   };
 
+  reject = () => {
+    this.setState(
+      (state) => {
+        const rejected = allCategories().reduce(
+          (acc, category) => ({
+            ...acc,
+            [category]: false,
+          }),
+          {} as Record<TCategory, false>
```

Comment:
I would not add type casting anywhere unless absolutely necessary. It can only bite: TS will not warn us if there's something else in the object because we're telling instead what it will look like.

Reply:
I agree, but I haven't found any other good way of dealing with `reduce`. If you know how to get it to work properly wrt type safety let me know

Reply:
I'm not sure what you're trying to accomplish specifically.